### ISSUE TITLE
Dropdown onChnage event fires twice?

### ISSUE THREAD

**Issue by VanquishedWombat** [fomantic/Fomantic-UI#1339]
Hi All - possible bug in dropdown where it fires too many onchange events. Please see fiddle, not there is already one onChange event fired on setup of the dropdown, and actual user selection of an option fires two onChange events. Is that how its meant to be ? 

https://jsfiddle.net/VanquishedWombat/4aepdsv6/29/

**Comment by prudho** [comment]
Yup, it's a bug... Removing the following part of HTML code fire only one event instead of two:
```html
<i class="search icon"></i>
  <span class="text">Seach by node attribute</span>
  <div class="menu">
</div>
```
Something's going wrong during initialization. Please note also that despite `fireOnInit` set to `false` (default setting), the `onChange` event is fired.

**Comment by tmikaeld** [comment]
I also noticed that if we have a multi-select, clearing the values will trigger an **onChange** while it's removing the entries, one trigger per entry.
